### DISCUSSION TITLE
Pre-recorded videos to tune config values

### DISCUSSION THREAD

**Discussion by Semmu** [Motion-Project/motion#1647] [Q&A]
Hi,

I'm thinking about pre-recording videos of:
* nothing happening (just wind blowing, sun moving, etc.)
* actual motion (people entering some area, cars, etc.)

Then using these pre-recorded videos and feeding them to motion to see how my current config at the time (e.g. threshold values) works.

The goal would be to find a set of config values that detects no motion for the "nothing happening" videos, but also actually detects motion in the "actual motion" videos.

This way one wouldn't have to test their config by e.g. walking in front of the camera after every little change, etc.

Has anyone done this type of tuning before?

Thanks in advance!

**Comment by tosiara** [comment]
That's a valid scenario. You can feed video as `file://` protocol in `netcam_url`

**Reply by Semmu** [reply]
Yeah, I was thinking exactly the same. Will give it a try!

**Comment by BrianThomas46** [comment]
Hi! I’m pointing a camcorder at a bird table/bath for 12+ hours at a time but going through the footage afterwards to find something worthwhile can be a bit tedious. (I do a lot of multicam work with FCPX but this latest idea is rather different.) I looked through most posts here and this seems to be a good thread within which to ask if "Motion" could help me. I'd much appreciate any advice and thanks in advance.
